libmscore/text.cpp

```diff
@@ -2659,7 +2659,12 @@ Element* Text::drop(const DropData& data)
                         }
                   else {
                         startEdit(data.view, data.pos);
-                        curLine().insert(_cursor, QChar(code));
+                        if (QChar::requiresSurrogates(code)) {
```

Comment:
I never stumbled upon this bug during earlier testing, because I never exercised Text::drop when not already in edit mode.  So similarly to earlier PR, need to insert both chars as surrogate pair.